library/Icinga/Web/LessCompiler.php

```diff
@@ -118,6 +125,10 @@ public function getLessFiles()
     {
         $lessFiles = $this->lessFiles;
 
+        if ($this->themeMode !== null) {
+            $lessFiles[] = $this->themeMode;
+        }
+
```

Comment:
Please put that at the very end so that it's applied **after** the theme. This will prevent that themes can override the mode's LESS variables.